lte/gateway/fabfile.py

```diff
@@ -875,7 +876,7 @@ def _run_integ_tests(gateway_ip='192.168.60.142', tests=None, federated_mode=Fal
         ' sudo ethtool --offload eth1 rx off tx off; sudo ethtool --offload eth2 rx off tx off;'
         ' source ~/build/python/bin/activate;'
         ' export GATEWAY_IP=%s;'
-        ' make -i %s enable-flaky-retry=true %s\''
+        ' make %s enable-flaky-retry=true %s\''
```

Comment:
Will not this be removing the continuous tests execution?

Reply:
Hi @ardzoht There were 2 ways to ignore failures and go for continuous execution. One was by passing this `-i` flag here in the make command itself and the 2nd one was by adding `-` before the command inside a makefile. This `-i` flag is working fine but creates problem in getting final failure status from the integ test run and for that reason artifact logs were also not published properly from VM. So, instead of this `-i`, I have added `-` inside the makefile for the commands when flaky retry is enabled. Please have a look on line numbers 60/77 in the Makefile changes in this PR where I have added `-` when we want to run integ test or fed integ test with flaky retry enabled. I have removed this `-i` flag from ci run but the `-i` flag is still valid after this change and if we want to run this make command manually with `-i`, that will work fine.